src/dataflow/source/kafka.rs

```diff
@@ -636,6 +646,14 @@ where
                                     ])
                                     .set(offset.offset);
 
+                                KAFKA_PARTITION_MESSAGE_INGESTED
+                                    .with_label_values(&[
+                                        &dp_info.topic_name,
+                                        &id.to_string(),
+                                        &partition.to_string(),
+                                    ])
+                                    .inc_by(1);
```

Comment:
this is fine, but you can just do `.inc()` for `inc_by(1)`

Reply:
lets fix that in a follow-on